src/components/VideoList/VideoList.spec.jsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import VideoList from './VideoList.component';
+
+const mockData = {
+  kind: 'youtube#searchListResponse',
+  etag: 'LRviZfd_p3HDDD2uBk5Qv7zaEQU',
+  nextPageToken: 'CBkQAA',
+  regionCode: 'MX',
+  pageInfo: {
+    totalResults: 2323,
+    resultsPerPage: 25,
+  },
+  items: [
+    {
+      kind: 'youtube#searchResult',
+      etag: '_PVKwNJf_qw9nukFeRFOtQ837o0',
+      id: {
+        kind: 'youtube#channel',
+        channelId: 'UCPGzT4wecuWM0BH9mPiulXg',
+      },
+      snippet: {
+        publishedAt: '2014-09-27T01:39:18Z',
+        channelId: 'UCPGzT4wecuWM0BH9mPiulXg',
```

Comment:
This mocked data is used in other tests, you can refactor this and eliminate the duplicated code